packages/nvidia_nat_core/src/nat/front_ends/fastapi/response_helpers.py

```
@@ -183,14 +175,17 @@ async def generate_streaming_response_full_as_str(payload: typing.Any,
+        async for item in generate_streaming_response_full(payload,
+                                                           session=session,
+                                                           streaming=streaming,
+                                                           result_type=result_type,
+                                                           output_type=output_type,
+                                                           filter_steps=filter_steps):
+            if (isinstance(item, ResponseIntermediateStep) or isinstance(item, ResponsePayloadOutput)):
+                yield item.get_stream_data()
+            else:
+                raise ValueError("Unexpected item type in stream. Expected ChatResponseSerializable, got: " +
+                                 str(type(item)))
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)